aggregator/src/datastore.rs

```diff
@@ -918,6 +918,30 @@ impl<C: Clock> Transaction<'_, C> {
         .transpose()
     }
 
+    #[cfg(feature = "test-util")]
+    pub async fn get_helper_client_reports_for_task(
```

Comment:
Q: why `helper`? I think this would work just as well for a leader task. This method also doesn't return reports, maybe `get_report_metadatas_for_task`?

Reply:
Oops, that was a leftover from when I was going to have this return something else and was dealing with the fact that the helper's `client_reports` rows don't have every column set, but `get_report_metadatas_for_task` is much better.